test/Datadog.Trace.IntegrationTests/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿#if NETFRAMEWORK
+namespace Datadog.Trace.IntegrationTests
+{
+    public static class Program
+    {
+        public static void Main()
```

Comment:
Do we need this?

Reply:
Unfortunately, yes. The build fails without it. I'm not entirely sure why, but likely todo with the `Microsoft.NET.Sdk.Web` SDK... 🤔

Reply:
Looks the web sdk has a different default output type. Adding `<OutputType>Library</OutputType>` to the project file should fix that.

Reply:
Ah yes, cheers. Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1269/commits/70859ec8a053b969f710706f43e7691a86c82037